openslides/core/migrations/0003_auto_20160803_1058.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.7 on 2016-08-03 10:58
+from __future__ import unicode_literals
+
+import uuid
+
+from django.db import migrations
+from openslides.core.models import Projector
+
+
+def add_speaker_overlay(apps, schema_editor):
+    """
+    Adds a list of speaker overlay to the existing projector id=1
+    """
+    projector = Projector.objects.get(id=1)
+    soverlay = {}
+    soverlay_uuid = uuid.uuid4().hex
+    soverlay[soverlay_uuid] = {
+        'name': 'core/speakeroverlay',
+        'stable' : True,
+        'uuid' : soverlay_uuid}
+    projector.elements.update(soverlay)
```

Comment:
The projector object (id=1) has the entry `elements` which is a dict of dicts, at the time of this call holding the entry for core/clock. I want to append the new entry (soverlay) and save the projector object. However, `/rest/core/projector/1` reports only core/clock as being part of the element. I guess I am confused by similarities/differences between Django, json and python.